Tchap/Modules/Room/Members/RoomParticipantsViewController.m

```diff
@@ -1335,7 +1351,14 @@ - (NSArray *)tableView:(UITableView *)tableView editActionsForRowAtIndexPath:(NS
 
 - (void)roomMemberDetailsViewController:(MXKRoomMemberDetailsViewController *)roomMemberDetailsViewController startChatWithMemberId:(NSString *)matrixId completion:(void (^)(void))completion
 {
-    [[AppDelegate theDelegate] createDirectChatWithUserId:matrixId completion:completion];
+    if (self.delegate && [self.delegate respondsToSelector:@selector(roomParticipantsViewController:startChatWithMemberId:completion:)])
```

Comment:
We could make `roomParticipantsViewController:startChatWithMemberId:completion` method mandatory in delegate definition as we will remove direct call to `AppDelegate`.